.github/workflows/push-new-version/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Push new version
+description: Commit and push the new version to the remote repository after the "Bump version" workflow has been completed.
```

Comment:
Please make it generic or find an existing action to make a git commit.

Reply:
done